android/src/main/java/com/qubit/reactnative/sdk/QubitSDKModule.java

```diff
@@ -139,18 +151,90 @@ public void getExperiences(ReadableArray experienceIds,
 
   @ReactMethod
   public void experienceShown(String callback) {
-    CallbackConnector callbackConnector = new CallbackConnectorImpl(callback, QubitSDK.getDeviceId());
+    ExperienceCallbackConnector callbackConnector = new ExperienceCallbackConnectorImpl(callback, QubitSDK.getDeviceId());
     callbackConnector.shown();
   }
 
+  @ReactMethod
+  public void getPlacement(
+      String placementId,
+      String mode,
+      String attributes,
+      String campaignId,
+      String experienceId,
+      Promise placementPromise
+  ) {
+    QubitSDK.getPlacement(
+        placementId,
+        matchMode(mode),
+        getAttributesJson(attributes),
+        new PlacementPreviewOptions(campaignId, experienceId),
+        placement -> {
+          WritableMap placementContentMap = WritableMapConverter.convertJsonToMap(placement.getContent());
```

Comment:
It doesn't match documentation of BridgeAPI.
According to BridgeAPI.md this function should return structure:
```
{
	  "data": {
	    "placementContent": {
	      "content": {
	        "image": "https://image.store.com/images/example.jpeg",
	        "message": "Hello World",
	        "url": "https://www.qubit.com"
	      },
	      "callbacks": {
	        "impression": "https://api.qubit.com/placements/callback?data=ggW4eyJtZXRhIjp7ImlkIjo",
	        "clickthrough": "https://api.qubit.com/placements/callback?data=mQW4eyJtZXRhIjp7Imlkx"
	      }
	    }
	  }
	}
```
But this method will return only:
```
{
	        "image": "https://image.store.com/images/example.jpeg",
	        "message": "Hello World",
	        "url": "https://www.qubit.com"
	      }
```

Reply:
Modified that logic so flat structure is returned:
```
 {
        "image": "https://image.store.com/images/example.jpeg",
        "message": "Hello World",
        "url": "https://www.qubit.com"
        "impressionUrl": "https://api.qubit.com/placements/callback?data=ggW4eyJtZXRhIjp7ImlkIjo",
        "clickthroughUrl": "https://api.qubit.com/placements/callback?data=mQW4eyJtZXRhIjp7Imlkx"
     }
```